Dockerfile.ci.s390x

```diff
@@ -0,0 +1,297 @@
+       "  local similar=\$(similar_installed_version \"\${req}\")\n" \
+       "  if test \"x\${similar}\" = \"x\"; then\n" \
+       "    return 1\n" \
+       "  fi\n" \
+       "  rbenv shell \"\${similar}\"\n" \
+       "}\n\n" \
+       "similar_installable_version() {\n" \
+       "  local req=\"\${1}\"\n" \
+       "  match_version \"\$(rbenv install -l)\" \"\${req}\"\n" \
+       "}\n\n" \
+       "install_all_rubies() {\n" \
+       "  for req in \"\${@}\"; do\n" \
+       "    if set_ruby_version \"\${req}\"; then\n" \
+       "      echo \"Skipping version \${req} as there is already version \$(rbenv version-name) installed\"\n" \
+       "      continue\n" \
+       "    fi\n" \
+       "    echo Installing \${req}\n" \
+       "    if ! RBENV_VERSION=\"\${req}\" update_env; then\n" \
+       "      echo \"Trying to pick up a similar version to \${req}\"\n" \
+       "      local new_version=\$(similar_installable_version \"\${req}\")\n" \
+       "      echo \"Picked up version \${new_version:-(default)}\"\n" \
+       "      RBENV_VERSION=\${new_version} update_env\n" \
+       "    fi\n" \
+       "  done\n" \
+       "}\n\n" \
+       "if [[ \"\${BASH_SOURCE[0]}\" = \"\${0}\" ]]; then\n" \
+       "  set -eo pipefail\n" \
+       "  shopt -s failglob\n\n" \
+       "  install_all_rubies \"\${@}\"\n" \
+       "fi" \
+       > ~/.local/bin/ruby_versions \
+    && chmod +x ~/.local/bin/ruby_versions \
+    && echo  "#!/bin/bash\n\n" \
+       "source ~/.bash_rbenv\n" \
+       "source \$(dirname \"\$(readlink -f \$0)\")/bundle_install_gemfiles\n\n" \
+       "bundle_install_all_rubies() {\n" \
+       "  for ruby_version in \$(rbenv whence ruby); do\n" \
+       "    echo \"Switching to \${ruby_version}\"\n" \
+       "    rbenv shell \"\${ruby_version}\"\n" \
+       "    gem update --system\n" \
+       "    (bundle_install_all_gemfiles | tee /tmp/bundle_install.log)\n" \
+       "    BUNDLER_REQUESTED=\"\$(sed -e '1,/Installed Bundler versions:/d' /tmp/bundle_install.log)\"\n" \
+       "    rm -f /tmp/bundle_install.log\n" \
+       "    echo \"Bundler versions installed: \${BUNDLER_REQUESTED[*]}\"\n" \
+       "    echo -n \"Bundler versions available on \$(ruby -v): \"\n" \
+       "    echo \"\$(gem list bundler | " \
+       "grep ^bundler | cut -d' ' -f2- | " \
+       "sed -e 's/(\(.*\))/\1/g' -e 's/,//g' -e 's/default:\s*//g')\"\n" \
+       "  done\n" \
+       "}\n\n" \
+       "if [[ \"\${BASH_SOURCE[0]}\" = \"\${0}\" ]]; then\n" \
+       "  set -eo pipefail\n" \
+       "  shopt -s failglob\n\n" \
+       "  bundle_install_all_rubies\n" \
+       "fi" \
+       > ~/.local/bin/bundle_install_rubies \
+    && chmod +x ~/.local/bin/bundle_install_rubies
+
+COPY .ruby-* /tmp/apisonator/
+RUN sudo chown -R "${USER_NAME}": /tmp/apisonator \
+    && cd /tmp/apisonator \
+    && rbenv_update_env
+
+# specify all versions to be installed, partial versions also understood
+ARG RUBY_VERSIONS="2.7.0"
```

Comment:
The original `Dockerfile.ci` file uses `MINOR.MAJOR` here to test against the latest patch release of MINOR.MAJOR, but this will pin the Ruby version to 2.7.0 - ie. it won't take 2.7.3, which is the original intention. Did you find any issues with this or otherwise preferred to pin the version to exactly 2.7.0?